Packs/Core/Playbooks/playbook-A_Successful_login_from_TOR.yml

```diff
@@ -0,0 +1,802 @@
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 331cade7-b3e9-4a82-8cf8-1ee613a71d7c
+    type: collection
+    task:
+      id: 331cade7-b3e9-4a82-8cf8-1ee613a71d7c
+      version: -1
+      name: Select containment & Eradication steps
+      type: collection
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "8"
+      - "19"
+      - "10"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Select containment steps on the user ${alert.username.[0]}
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: multiSelect
+        options: []
+        optionsarg:
+        - {}
+        - simple: Disable the user account
+        - simple: Expire the user password (Active Directory Only)
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      - id: "1"
+        label: ""
+        labelarg:
+          simple: "Would you like to block traffic from TOR exit nodes using PAN-OS and Palo Alto Networks predefined EDL. \nNOTICE: By selecting \"Yes, commit automatically\" you are allowing to automatically commit the rule to your firewalls."
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - simple: "No"
+        - simple: Yes, commit manually
+        - simple: Yes, commit automatically
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Select containment & Eradication steps
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 9e1167c7-f6f3-4230-8db0-ef2f65c915b7
+    type: condition
+    task:
+      id: 9e1167c7-f6f3-4230-8db0-ef2f65c915b7
+      version: -1
+      name: User Exists And Disable User Account Selected?
```

Comment:
Disable Selected User?

Reply:
But its not "selected user" since there will be only one user. The analyst selecting action not user

Reply:
Changed it to "Disable User Account"